static/styles/message_recipient_bar.css

```diff
@@ -0,0 +1,216 @@
+:root {
+    color-scheme: light;
```

Comment:
This duplicates a rule in `zulip.css` and doesn’t belong here.

Reply:
Okay.